flutter_cache_manager/example/pubspec.yaml

```
@@ -5,30 +5,24 @@ publish_to: none
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.10.2 <3.0.0"
+  sdk: ">=3.0.0 <4.0.0"
```

We can support 2.12.0 and higher in the example
```suggestion
  sdk: ">=2.12.0 <4.0.0"
```